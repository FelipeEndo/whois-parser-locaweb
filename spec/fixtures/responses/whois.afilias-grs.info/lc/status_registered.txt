Domain Name: NIC.LC
Registry Domain ID: D105800000000946482-AGRS
Registrar WHOIS Server:
Registrar URL:
Updated Date: 2020-12-08T22:22:29Z
Creation Date: 2002-12-08T00:00:00Z
Registry Expiry Date: 2021-12-08T00:00:00Z
Registrar Registration Expiration Date:
Registrar: NicLc Registrar
Registrar IANA ID: 119
Registrar Abuse Contact Email:
Registrar Abuse Contact Phone:
Reseller:
Domain Status: ok https://icann.org/epp#ok
Registrant Organization: Nic LC
Registrant State/Province: Not Provided
Registrant Country: LC
Name Server: NS1.NIC.AG
Name Server: NS.PATRICKLAY.COM
DNSSEC: unsigned

>>> Last update of WHOIS database: 2021-11-09T15:15:18Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy.
The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
