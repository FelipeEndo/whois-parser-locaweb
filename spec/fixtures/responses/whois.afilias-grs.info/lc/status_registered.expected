#disclaimer
  %s == "Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy. The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."


#domain
  %s == "nic.lc"

#domain_id
  %s == "D105800000000946482-AGRS"


#status
  %s == ["ok https://icann.org/epp#ok"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2002-12-08 00:00:00 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2020-12-08 22:22:29 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2021-12-08 00:00:00 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "119"
  %s.name         == "NicLc Registrar"
  %s.url          == ""

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type         == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id           == nil
  %s[0].name         == nil
  %s[0].organization == "Nic LC"
  %s[0].address      == nil
  %s[0].city         == nil
  %s[0].zip          == nil
  %s[0].state        == "Not Provided"
  %s[0].country_code == "LC"
  %s[0].phone        == nil
  %s[0].fax          == nil
  %s[0].email        == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{0}

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{0}

#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.nic.ag"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns.patricklay.com"
